lib/jobs/sftp-job.js

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+module.exports = sftpJobFactory;
+
+var Client = require('ssh2').Client;
```

Comment:
Since you add the `ssh2` into the injectableTable (see https://github.com/RackHD/on-tasks/pull/163/files#diff-168726dbe96b3ce427e7fedce31bb0bcR11), so you'd better to use it using `di`